packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.stories.js

```diff
@@ -0,0 +1,23 @@
+// /* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, number } from "@storybook/addon-knobs";
+
+import SfQuantitySelector from "./SfQuantitySelector.vue";
+
+storiesOf("Atoms|QuantitySelector", module)
+  .addDecorator(withKnobs)
+  .add("Common", () => ({
+    components: { SfQuantitySelector },
+    props: {
+      qty: {
+        default: number("qty", 0, "Props")
```

Comment:
```suggestion
        default: number("qty", 0, {}, "Props")
```
Please see conversation at https://github.com/DivanteLtd/storefront-ui/pull/485#discussion_r354696553